src/content/en/fundamentals/performance/rail.md

```diff
@@ -78,18 +83,24 @@ For actions that take longer than 500ms to complete, always provide feedback.
 
 Success: Respond to user's touchmoves and scrolling in under 16ms.
```

Comment:
✔️ 

I think the `Success` header was mistakenly created when we migrated the site content to its latest infrastructure. Also, yes, this didn't make sense here, so I deleted it